src/__stories__/Misc/Toaster.tsx

```diff
@@ -0,0 +1,16 @@
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+import { ToastType } from 'react-toastify';
+import { Box } from '../../Box';
+import { toast, Toaster } from '../../Toaster';
+
+storiesOf('Miscellaneous:Toaster', module).add('with defaults', () => (
```

Comment:
Would slice the story to 2-3 and make type prop a knob.

Reply:
:+1: added knobs and simplified to a single story.